examples/active.ini

```diff
@@ -25,11 +25,12 @@ guessed_mhz_to_ghz = 1.5167
 
 [odnp_params]
 max_power = 3.16
-power_steps = 4
+power_steps = 14
 num_t1s = 2
 uw_dip_center_ghz = 9.820639
 uw_dip_width_ghz = 0.02
 fir_rep = 500000.0
+guessed_phalf = 0.5
```

Comment:
I don't agree with having either of these here.  This file should have a bare minimum of parameters committed.  Any defaults should be defined above (see where I ask about `None` --> why not just give a default?)

Reply:
but I will move phalf to sample parameters instead since it is dependent on concentration

Reply:
200 μM and 9.4 mM are vastly different types of samples.  Based on that range, I think that 0.3 probably a reasonable guess.  See previous comment as to where this goes.

Reply:
but I think it's not clear -- there should be no reference to this parameter in the committed version of active.ini, because the committed version of active.ini should be absolutely bare bones.

Reply:
the default should be given in `SpinCore_pp/config_parser_fn.py`